pkg/controller/admissionchecks/provisioning/admissioncheck_reconciler.go

```diff
@@ -61,7 +64,11 @@ func (a *acReconciler) Reconcile(ctx context.Context, req reconcile.Request) (re
 
 	if currentCondition.Status != newCondition.Status {
 		apimeta.SetStatusCondition(&ac.Status.Conditions, newCondition)
-		return reconcile.Result{}, a.client.Status().Update(ctx, ac)
+		if err := a.client.Status().Update(ctx, ac); err != nil {
+			return reconcile.Result{}, err
+		}
+		a.record.Eventf(ac, corev1.EventTypeNormal, "UpdatedAdmissionChecks", "Admission checks status is changed to %s", newCondition.Status)
```

Comment:
I'm not sure about the usefulness of this event. Typically, an administrator doesn't need to pay attention to changes in this object.
If they happen to need more detail, inspecting the AdmissionCheck status should be enough.

Reply:
removed https://github.com/kubernetes-sigs/kueue/pull/1271/commits/3b15bd9a0ad460731328c4e0eaeb91897f09cfb0